schema/common/defaults/configuration/opensearch-dashboards.yml

```diff
@@ -0,0 +1,11 @@
+kind: configuration/opensearch-dashboards
+title: "OpenSearch-Dashboards"
+name: default
+specification:
+  dashboards_os_user: opensearchdboard
```

Comment:
Specify in variable that this is an os user? Maybe also separate group?

Reply:
ad a)
ok
Implemented now.
ad b)
As discussed before: there are two approaches and it's hard to say which is better.
Probably would be good to discuss it within the team before changing.